### ISSUE TITLE
Whitelist Pi Network application

### ISSUE THREAD

**Issue by b-derouet** [ph00lt0/blocklist#16]
**Tool used (pick one)**:
 - [ ] uBlock Origin
 - [ ] Adguard for iOS
 - [x] Adguard for Android
 - [ ] Adguard Browser add-on/extension
 - [ ] Brave Ad Block
 - [ ] PiHole
 - [ ] Adguard Home

[comment]: <> (Change one of the boxes to - [x] to "select" it.)

**What service are you trying to use?'**
Applications of Pi Network ecosystem.

**What does not work?**
Applications can't connect to the main application since the URL is blocked. It is needed to establish a connection with the Pi API (authentication, transactions).

**Which rule you believe is causing this?**
app-cdn.minepi.com^

**Comment by b-derouet** [comment]
Sorry but I am not part of the Core Team nor do I work for them.

**Comment by b-derouet** [comment]
But I think it's too late now that you've made an overly hasty description of my profile...

**Comment by b-derouet** [comment]
1. Your profile seems to be associated with the project, I don't see much else activity (as you also say yourself)

As a student and co-founder of a company, I mainly work on the private gitlab instances of my university and my company. For my Github profile, I don't have many projects outside of school and work for the moment.

2. The main domain of the website links to socialchain[.]app a website flagged by VirusTotal

As can be seen from the VirusTotal scan, only two security vendors have flagged the URL, the community score is 0, which in itself is not that negative.

3. Many sources flagged this project as a scam

As I said above, I understand that some people see this project as a scan as it aims to become a real ecosystem based around a blockchain and that uninformed users who only see this project as a way to make a quick buck are disappointed that they can't sell their crypto for fiat currency or other crypto-currencies right away. But easy money is not the goal of the project, the Core Team made the choice to develop the utility of the crypto-currency before its value. It's good if the currency has value, but it's better if you can actually use it. In my opinion, it would have been much more dubious to list the crypto-currency on exchanges so that it has value right away (yes, it is not officially listed yet, what you can find on the internet on exchanges are only dirty IOUs, promises to buy/sell, but not authorized by the Core Team).

4. Many vague blog websites with not much content nor reputation praise this project coming across as SEO poisoning.

The Core Team recently announced (for Pi Day) that the community now includes 45 million active people worldwide, this does not include inactive/deleted accounts (which would increase this number much more). Many of the pioneers still think this is an easy money making project and therefore advertise the project massively based on this hope of a quick buck (you can't blame them, the lure of money does a lot of things). Whether it's the pioneers looking to make easy money while doing bad publicity, or the former pioneers disappointed to have followed the publicity only to find out that the project was not at all about easy money, it's the community and the project that unfortunately takes a hit.

5. The domainname is blocked by several blocklists: https://rethinkdns.com/search?q=minepi.com

If you summarise everything I said above, it is unfortunately understandable, but I can always use Brandolini's law to conclude:
> The amount of energy needed to refute bullshit is an order of magnitude bigger than that needed to produce it.

I'm not here to say that these are good arguments, just that this is my view of the matter and that I can of course be wrong.

**Comment by b-derouet** [comment]
Well, I tried, thanks anyway!
